10.9-libcxx/stable/main/finkinfo/libs/pythonmods/webencodings-py.info

```diff
@@ -0,0 +1,49 @@
+# -*- coding: ascii; tab-width: 4 -*-
+Info3: <<
+
+Package: webencodings-py%type_pkg[python]
+Version: 0.5.1
+Revision: 1
+Maintainer: Derek Homeier <dhomeie@gwdg.de>
+License: BSD
+Type: python (2.7 3.4 3.5 3.6)
+Homepage: https://github.com/SimonSapin/python-webencodings
+Source: https://pypi.io/packages/source/w/webencodings/webencodings-%v.tar.gz
+Source-MD5: 32f6e261d52e57bf7e1c4d41546d15b8
+
+Depends: python%type_pkg[python]-shlibs
+BuildDepends: python%type_pkg[python], setuptools-tng-py%type_pkg[python]
+
+Description: Character encoding aliases for legacy pages
+
+DescDetail: <<
+This is a Python implementation of the WHATWG Encoding standard.
```

Comment:
```
Validating package file /sw/fink/dists/local/distributions/10.9-libcxx/stable/main/finkinfo/libs/pythonmods/webencodings-py.info...
Warning: "descdetail" contains line(s) exceeding 79 characters. (webencodings-py.info)
```

Reply:
Should learn not to make "trivial" changes after building with '-m'...